README.md

```diff
@@ -1,8 +1,10 @@
-<h2><img align="center" src="https://github.com/NVIDIA-BioNeMo-blueprints/generative-virtual-screening/blob/main/nvidia-logo.png?raw=true">NVIDIA BioNeMo Blueprint: Generative Virtual Screening for Drug Discovery</h2>
+<h2><img align="center" src="https://github.com/NVIDIA-NIM-Agent-Blueprints/generative-virtual-screening/blob/main/nvidia-logo.png?raw=true">NVIDIA NIM Agent Blueprints: Generative Virtual Screening for Drug Discovery</h2>
+
+[![Open In AI Workbench](https://img.shields.io/badge/Open_In-AI_Workbench-76B900)](https://ngc.nvidia.com/open-ai-workbench/aHR0cHM6Ly9naXRodWIuY29tL05WSURJQS1CaW9OZU1vLWJsdWVwcmludHMvZ2VuZXJhdGl2ZS12aXJ0dWFsLXNjcmVlbmluZw==)
 
 ![Generative Virtual Screening for Drug Discovery-r2 (1)](https://github.com/user-attachments/assets/dbb1795a-7e3f-4363-9b20-4bc4d67d04bb)
 
-The NVIDIA BioNeMo Blueprint for generative virtual screening shows how generative AI and accelerated NIM microservices can be used to design optimized small molecules smarter and faster.
+The NVIDIA NIM Agent Blueprints for generative virtual screening shows how generative AI and accelerated NIM microservices can be used to design optimized small molecules smarter and faster.
```

Comment:
Should this be "NIM Agent Blueprint" singular?  Also in the heading above?

Reply:
My copy of the readme is fairly dated, I'll defer to the language used in the current main branch.

Reply:
fixed in most recent commit